_ont/ont-huawei-ma5671a-root-web.md

```diff
@@ -41,9 +41,7 @@ After this is done, reboot the stick, after connecting it to the router via an e
 
 {% include alert.html content="If your subnet is `192.168.1.0/24` make sure you have no ip conflicts." alert="Note" icon="svg-warning" color="yellow" %}
 
-{% include alert.html content="Make sure to disable SFP TX fault detection, otherwise the RX loss will prevent you from connecting to the mini SFP ONT at this point. Don't simply attach the fiber cable to work around this issue as the OLT may ban you." alert="Note" icon="svg-warning" color="yellow" %}
-
-{% include alert.html content="In media converters and on MikroTik it is not necessary to disable TX fault." alert="Info" icon="svg-info" color="blue" %}
+{% include alert.html content="On some SFP host devices you might not be able to connect to the stick if there's no optical signal, in that case you need to connect the fiber to make changes on the stick" alert="Note" icon="svg-warning" color="yellow" %}
```

Comment:
I think the message can also be put back on some mikrotiks such as the RB5009 and CRS305, but on others like as the RB4001 it is needed